### DISCUSSION TITLE
Suppress Script Output Option

### DISCUSSION THREAD

**Discussion by btyl** [usnistgov/macos_security#300] [Ideas] [answered]
An option to be able to suppress script output so that it is not captured in Jamf policy logs.  We have a very large Mac population and the output from the script that gets captured in the policy logs is greatly increasing the size of the policy logs table in the database.  Even better would be just to output any failed controls so those can be captured in the policy logs.

**Comment by robertgendler** [comment]
I'm sure there's probably a way? But I'm totally unsure how to do this since there are tons of echos in the compliance script. You'd want SOME output just not ALL the output.

**Comment by brodjieski** [comment, marked as answer]
@btyl there is a new branch to test with... review the changes in this commit: https://github.com/usnistgov/macos_security/commit/7778fe30cd374329e7314f6822f1b2a551946e91
